MediaBrowser.Controller/IResourceFileManager.cs

```diff
@@ -1,16 +1,7 @@
-using System;
-using System.IO;
-using System.Threading.Tasks;
-using MediaBrowser.Model.Services;
-
 namespace MediaBrowser.Controller
 {
     public interface IResourceFileManager
     {
-        Task<object> GetStaticFileResult(IRequest request, string basePath, string virtualPath, string contentType, TimeSpan? cacheDuration);
-
-        Stream GetResourceFileStream(string basePath, string virtualPath);
-
-        string ReadAllText(string basePath, string virtualPath);
+        string GetResourcePath(string basePath, string virtualPath);
```

Comment:
This might break some plugins if they implement this, not sure I've seen it though.